recipes/r-fake/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set version = '1.3.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fake
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fake_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fake/fake_{{ version }}.tar.gz
+  sha256: 07412cfb614e6e801437ed71a35f4acf7fdc3b898fdc14fec2b5df84111ef426
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-mass
+    - r-rdpack
+    - r-huge
+    - r-igraph
+    - r-withr >=2.4.0
+  run:
+    - r-base
+    - r-mass
+    - r-rdpack
+    - r-huge
+    - r-igraph
+    - r-withr >=2.4.0
+
+test:
+  commands:
+    - $R -e "library('fake')"           # [not win]
+    - "\"%R%\" -e \"library('fake')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=fake
+  license: GPL-3
+  summary: This R package can be used to generate artificial data conditionally on pre-specified
+    (simulated or user-defined) relationships between the variables and/or observations.
+    Each observation is drawn from a multivariate Normal distribution where the mean
+    vector and covariance matrix reflect the desired relationships. Outputs can be used
+    to evaluate the performances of variable selection, graphical modelling, or clustering
+    approaches by comparing the true and estimated structures (B Bodinier et al (2021)
+    <arXiv:2106.02521>).
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - barbarabodinier
```

please also add the conda-forge r maintainer group:
```suggestion
    - conda-forge/r
    - barbarabodinier
```